web/src/test/java/com/graphhopper/http/BaseServletTester.java

```diff
@@ -69,9 +69,6 @@ public void setUpGuice(Module... modules) {
      * This method will start jetty with andorra area loaded as OSM.
      */
     public void setUpJetty(CmdArgs args) {
-        if (injector != null)
-            throw new UnsupportedOperationException("Do not call guice before");
-
```

Comment:
Had to remove this, otherwise it wouldn't have been possible to call setupGuice.

Reply:
It should be possible. I'll have a look.

Reply:
Ah, nice, that makes sense, thanks :).